internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-collectd-kafka-broker.discovery.yaml.tmpl

```diff
@@ -0,0 +1,43 @@
+{{ receiver "smartagent/collectd/kafka_broker" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)kafka.*broker"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)kafka.*broker"
+    k8s_observer: type == "port" and pod.name matches "(?i)kafka.*broker"
+  config:
+    default:
+      type: collectd/kafka
+      extraMetrics: ["*"]
```

Comment:
We don't want to enabled all metrics by default since many aren't categorized (custom w/ billing implications).